core/src/test/java/test/org/apache/spark/Java8RDDAPISuite.java

```diff
@@ -246,7 +246,7 @@ public void mapPartitions() {
 
   @Test
   public void sequenceFile() throws IOException {
-    File tempDir = JavaUtils.createTempDir();
+    File tempDir = Utils.createTempDir(System.getProperty("java.io.tmpdir"), "spark");
```

Comment:
in core module, change to call `Utils.createTempDir`

Reply:
I think this is OK; we can also add an overload of createTempDir so that Java can call this without repeating the arguments, but it's not a big deal

Reply:
done